docs/topics/gradle/gradle-configure-project.md

```diff
@@ -512,6 +523,74 @@ If you do not need a standard library at all, you can add the opt-out option to
 kotlin.stdlib.default.dependency=false
 ```
 
+#### Versions alignment of transitive dependencies
+
+If you explicitly write the Kotlin version 1.8.0 or higher in your dependencies, for example: 
+`implementation("org.jetbrains.kotlin:kotlin-stdlib:1.8.0")`, then the Kotlin Gradle Plugin uses this Kotlin version 
+for transitive `kotlin-stdlib-jdk7` and `kotlin-stdlib-jdk8` dependencies. This is for avoiding class duplication from 
+different stdlib versions (learn more about merging `kotlin-stdlib-jdk7` and `kotlin-stdlib-jdk8` into `kotlin-stdlib`(whatsnew18.md#updated-jvm-compilation-target)). 
+You can disable this behavior with the `kotlin.stdlib.jdk.variants.version.alignment` Gradle property:
+
+```properties
+ `kotlin.stdlib.jdk.variants.version.alignment=false`
+```
+
+##### Other ways to align versions {initial-collapse-state="collapsed"}
+
+* In case you don't use the Kotlin Gradle plugin or have issues with versions alignment, align all versions 
+  via the Kotlin [BOM](https://docs.gradle.org/current/userguide/platforms.html#sub:bom_import). 
+  Declare a platform dependency on `kotlin-bom` in your build script:
+
+  ```kotlin
+  implementation(platform("org.jetbrains.kotlin:kotlin-bom:%kotlinVersion%"))
+  ```
+
+* If you don't have a standard library explicitly: `kotlin.stdlib.default.dependency=false` in your `gradle.properties`,
+  but one of your dependencies transitively brings some old Kotlin stdlib version, for example, `kotlin-stdlib-jdk7:1.7.20` and 
+  another dependency transitively brings `kotlin-stdlib:1.8+` – in this case, you can require `1.8.0` versions of these
```

Comment:
Will it be ok to replace it with a variable kotlinVersion, which later will have values `1.8.10`, `1.9.0`, and so on?

Reply:
I think - yes